front_end/src/components/landing_page/Landing.js

```diff
@@ -17,15 +37,165 @@ class Landing extends Component {
   };
 
   render() {
-    const { classes } = this.props;
+    let styles = {
+      root: {
+        width: "auto",
+        flexGrow: 1,
+        marginLeft: 150,
+        marginRight: 150
+      },
+      imgSlideShow: {
+        display: "flex",
+        backgroundImage: `url(${SF})`,
+        backgroundSize: "cover",
+        maxWidth: "100%",
+        maxHeight: "100%",
+        height: "70vh",
+        minHeight: 450
+      },
+      searchWidgetBox: {
+        alignSelf: "center",
+        background: "rgba(0,0,0,0.5)",
+        padding: 20,
+        width: "100%"
+      },
+      seperator: {
+        height: 150,
+        width: 300
+      },
+      cityCollage: {
+        width: "auto",
+        marginLeft: 150,
+        marginRight: 150
+      },
+      collageImg: {
+        height: "100%"
+      }
+    };
+
+    function dealStyle(img) {
+      return {
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundImage: `url(${img})`,
+        backgroundSize: "cover",
+        width: 450,
+        height: 300
+      };
+    }
 
-    return <div className="landing" />;
+    return (
+      <div
+        style={{
```

Comment:
Can you move these css out of return ( ) by using const or .css ?

Reply:
Done